test/JDBC/expected/test_db_collation-vu-verify.out

```diff
@@ -0,0 +1,1800 @@
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db11', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1_ci_as
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db121', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db122', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db131', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1250_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db132', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1257_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db133', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db134', 'Collation');
+GO
+~~START~~
+sql_variant
+estonian_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db135', 'Collation');
+GO
+~~START~~
+sql_variant
+finnish_swedish_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db136', 'Collation');
+GO
+~~START~~
+sql_variant
+french_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db137', 'Collation');
+GO
+~~START~~
+sql_variant
+modern_spanish_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db138', 'Collation');
+GO
+~~START~~
+sql_variant
+polish_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db139', 'Collation');
+GO
+~~START~~
+sql_variant
+traditional_spanish_ci_ai
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db141', 'Collation');
+GO
+~~START~~
+sql_variant
+bbf_unicode_cp1_ci_as
+~~END~~
+
+
+SELECT DATABASEPROPERTYEX('test_db_collation_vu_prepare_db142', 'Collation');
+GO
+~~START~~
+sql_variant
+<NULL>
+~~END~~
+
+
+-- Case 4: Verify that the columns are picking correct collation (IMPLICIT COLLATION)
+-- Case 7: Validate collations of datatypes using catalogs like sys.types
+-- Case 8: CREATE custom datatype from system datatype and validate the same using sys.types
+USE test_db_collation_vu_prepare_db11;
+GO
+
+SELECT name, collation_name from sys.columns WHERE name IN ('c', 'nc', 'v', 'nv', 't', 'nt', 'sqlv', 'sn') ORDER BY name + COALESCE(collation_name, '');
+GO
+~~START~~
+varchar#!#varchar
+c#!#bbf_unicode_cp1_ci_ai
+c#!#bbf_unicode_cp1_ci_as
+nc#!#bbf_unicode_cp1_ci_ai
+nc#!#bbf_unicode_cp1_ci_as
+nt#!#bbf_unicode_cp1_ci_ai
+nt#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_as
+~~END~~
```

Comment:
Is it possible to get table name also to avoid confusion?

Reply:
Updated